src/components/ProofOfAddress/Guidance/graphic.js

```diff
@@ -0,0 +1,31 @@
+import { h } from 'preact'
+import style from './style.css'
+
+export default () => {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 272 217" preserveAspectRatio="xMinYMin meet" className={style.graphic}>
```

Comment:
Why do we need to do this? Can this svg be optimised?

Reply:
Some browsers are not rendering `<text>` elements with the correct font. To avoid having to convert text to outlines to make sure appearance is preserved, I am using the DOM to render this element.

Reply:
I see. Good to know 👍